src/lib/Components/ArtworkGrids/InfiniteScrollGrid.tsx

```diff
@@ -258,26 +259,11 @@ class InfiniteScrollArtworksGrid extends React.Component<Props, State> {
     return sections
   }
 
-  // Lifted pretty much straight from RN’s ListView.js
-  onScroll = event => {
-    const scrollProperties = event.nativeEvent
-    const contentLength = scrollProperties.contentSize.height
-    if (contentLength !== this.sentEndForContentLength) {
-      const offset = scrollProperties.contentOffset.y
-      const visibleLength = scrollProperties.layoutMeasurement.height
-      const distanceFromEnd = contentLength - visibleLength - offset
-      if (distanceFromEnd < PageEndThreshold) {
-        this.sentEndForContentLength = contentLength
-        this.fetchNextPage()
-      }
-    }
-  }
-
   render() {
     const artworks = this.state.sectionDimension ? this.renderSections() : null
     return (
       <ScrollView
-        onScroll={this.onScroll}
+        onScroll={isCloseToBottom(this.fetchNextPage)}
```

Comment:
Updated callback with the new `isCloseToBottom` util by moving logic from above

Reply:
👍

Is the same threshold being used as above, though? (And it also seems like the threshold constant can be removed from this file?)

Reply:
Good catch -- removed the additional const